ui/DesignSystem/Primitive/Input/Text.svelte

```diff
@@ -156,6 +173,8 @@
       <div class="validation-row">
         <p>{validation.message}</p>
       </div>
+    {:else if hint !== null && (value === '' || value === null)}
+      <p class="typo-text hint">{hint}</p>
```

Comment:
This duplicated what I've done on [line 156](https://github.com/radicle-dev/radicle-upstream/pull/914/files#diff-10dae02f6dfcb8b7213057c861dde888R156). I didn't know how else to match that it's not being validated, without duplicating what's already on line 160 - 175. Open to suggestions

Reply:
If we type this component, the truthiness could be easier to check, e.g.
```typescript
  export let hint = "";
  export let value = "";

  ...

  $: showHint = hint && !value.length && !validation
```

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/914/commits/719f4ac0b9e8a9e1b1886073bf1c520b76fa5dc4